### DISCUSSION TITLE
How to config dual stick in Star Wars Racer Episode 1

### DISCUSSION THREAD

**Discussion by Spacejockey-GH** [darthcloud/BlueRetro#93] [Q&A]
I have tried in various ways but have had no luck. I tried assigning a device 1 button to be assigned to output 2 but the game never recognizes that there are two controllers connected. If I connected output 2 to port 1 of the console the controller is detected.

**Comment by darthcloud** [comment]
before getting into the mapping, is your 2nd port detect in any other game multiplayer game?

**Reply by Spacejockey-GH** [reply]
I just tried Pokemon stadium and it detects two connected controllers

**Reply by darthcloud** [reply]
Will try later today and report back.

**Reply by Spacejockey-GH** [reply]
IT´S WORKING!!!
My bad, 
If you put a controller in port 1 and another in port 2 of the console, the option "2 VS" appears
But if you put one in port 1 and another in port 3 (it is one of the conditions for the dual stick mode cheat) that option does not appear ("2VS")

**Reply by darthcloud** [reply]
Glad you got it working!!
